packages/prettier-plugin-typespec/ThirdPartyNotices.txt

```diff
@@ -7,19 +7,3 @@ This project incorporates components from the projects listed below. The
 original copyright notices and the licenses under which Microsoft received such
 components are set forth below. Microsoft reserves all rights not expressly
 granted herein, whether by implication, estoppel or otherwise.
-
```

Comment:
This is generated at build time now?

Reply:
no it is just accurate now, we don't bundle prettier in the plugin